<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа 4 - Рудь, ИТ-621м</string>
  </property>
  <widget class="QPushButton" name="callButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Позвонить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerCallButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять </string>
   </property>
  </widget>
  <widget class="QPushButton" name="endCallButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сбросить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToBalanceButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Пополнить на 50 у.е.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="currentState">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Баланс:</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Последнее состояние</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>История состояний</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
